pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -294,10 +295,13 @@ func linkPythonPackage(ws pkgWorkspace.Context, root string, pkg *schema.Package
 			return fmt.Errorf("error opening requirments.txt: %w", err)
 		}
 
-		fBytes = []byte(packageSpecifier + "\n" + string(fBytes))
-		err = os.WriteFile(fPath, fBytes, 0o600)
-		if err != nil {
-			return fmt.Errorf("could not write requirments: %w", err)
+		lines := strings.Split(string(fBytes), "\n")
```

Comment:
I know the code was previously always using `\n` as the line separator, but what about on Windows where the line endings of the file could be `\r\n`?

Reply:
Good call, updated